sqlite/Cargo.toml

```diff
@@ -23,7 +23,7 @@ serde = ["deadpool/serde", "serde_1"]
 [dependencies]
 deadpool = { path = "../", version = "0.9.1", default-features = false, features = ["managed"] }
 deadpool-sync = { path = "../sync", version = "0.1" }
-rusqlite = "0.25"
+rusqlite = "0.28"
```

Comment:
Please remove this change from the PR.

I'll merge https://github.com/bikeshedder/deadpool/tree/feature/rusqlite-027 separately (so it becomes part of the main branch) and then bump the `rusqlite` version to 0.28 with a separate commit. :+1:

Reply:
I will revert the upgrade once I know I've fixed the CI workflow.

Reply:
Gotcha. Thanks a lot for working on this. :smiley:

Reply:
Could you trigger the CI build, please?
I think because I'm an outside contributor it does not trigger the workflow and you need to allow it.

Reply:
For some reason I don't see any option for running the CI pipeline. Other PRs have shown this option in the past. In this case I don't get an option to do so.

Reply:
I tried adding a manual approval, but it still doesn't show the dialog asking if the CI pipeline should be run.

Reply:
Maybe it's related to this PR making changes to the CI configuration? I can see that the pipeline was run on your own clone of the repository: https://github.com/elpiel/deadpool/actions

Reply:
Oh... that might be the reason?

![image](https://user-images.githubusercontent.com/1112569/179354441-2b60abc3-7be8-4f28-b541-08a307ef35e9.png)

Reply:
totally my bad :sweat_smile:, I did a force push. Let's see how it goes.